modules/textures/src/lib/parsers/parse-dds.js

```diff
@@ -64,19 +64,24 @@ export function parseDDS(data) {
   });
 }
 
-function int32ToFourCC(value) {
-  return String.fromCharCode(
-    value & 0xff,
-    (value >> 8) & 0xff,
-    (value >> 16) & 0xff,
-    (value >> 24) & 0xff
-  );
-}
-
 export function getDxt1LevelSize(width, height) {
   return ((width + 3) >> 2) * ((height + 3) >> 2) * 8;
 }
 
 export function getDxtXLevelSize(width, height) {
   return ((width + 3) >> 2) * ((height + 3) >> 2) * 16;
 }
+
+/**
+ * Convert every byte of Int32 value to char
+ * @param {number} value - Int32 number
+ * @returns {string} string of 4 characters
+ */
+function _int32ToFourCC(value) {
```

Comment:
We normally don't use underscore on private functions as the lack of an export keyword is enough.